cmd/analyze-xhprof.go

```diff
@@ -14,15 +12,17 @@ func init() {
 	RootCmd.AddCommand(xhprofCmd)
 	xhprofCmd.Flags().StringVarP(&field, "field", "f", "excl_wt", "Field to view/sort (wt, excl_wt, cpu, excl_cpu, memory, excl_memory, io, excl_io)")
 	xhprofCmd.Flags().Float32VarP(&minPercent, "min", "m", 1, "Display items having minimum percentage (default 1%) of --field, with respect to main()")
```

Comment:
Escape percent with `%%`

Reply:
This would show two percent signs. `%` is not parsed specially here.